src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(.\fonts\Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: bold;
+}
+
+.form {
+  position: relative;
```

Comment:
Input will broke if u add some element to top of the form, let's add relative to the input wrapper